war/src/main/webapp/scripts/behavior.js

```diff
@@ -20,60 +20,74 @@
             }
         });
 
-	// Call Behaviour.apply() to re-apply the rules (if you
-	// update the dom, etc).
+   // Call Behaviour.apply() to re-apply the rules (if you
+   // update the dom, etc).
 
    License:
 
-   	This file is entirely BSD licensed.
+        This file is entirely BSD licensed.
 
    More information:
 
-   	http://ripcord.co.nz/behaviour/
+        http://ripcord.co.nz/behaviour/
 
 */
 
-var Behaviour = (function() {
-    var storage = [{selector: '', id: '_deprecated', priority: 0}];
-    return {
-
+var Behaviour = (function () {
+  var storage = [{ selector: "", id: "_deprecated", priority: 0 }];
+  return {
     /**
      * Specifies something to do when an element matching a CSS selector is encountered.
      * @param {String} selector a CSS selector triggering your behavior
      * @param {String} id combined with selector, uniquely identifies this behavior; prevents duplicate registrations
      * @param {Number} priority relative position of this behavior in case multiple apply to a given element; lower numbers applied first (sorted by id then selector in case of tie); choose 0 if you do not care
      * @param {Function} behavior callback function taking one parameter, a (DOM) {@link Element}, and returning void
      */
-    specify : function(selector, id, priority, behavior) {
-        for (var i = 0; i < storage.length; i++) {
-            if (storage[i].selector == selector && storage[i].id == id) {
-                storage.splice(i, 1);
-                break;
-            }
+    specify: function (selector, id, priority, behavior) {
+      for (var i = 0; i < storage.length; i++) {
+        if (storage[i].selector == selector && storage[i].id == id) {
+          storage.splice(i, 1);
+          break;
         }
-        storage.push({selector: selector, id: id, priority: priority, behavior: behavior});
-        storage.sort(function(a, b) {
-            var location = a.priority - b.priority;
-            return location != 0 ? location : a.id < b.id ? -1 : a.id > b.id ? 1 : a.selector < b.selector ? -1 : a.selector > b.selector ? 1 : 0;
-        });
+      }
+      storage.push({
+        selector: selector,
+        id: id,
+        priority: priority,
+        behavior: behavior,
+      });
+      storage.sort(function (a, b) {
+        var location = a.priority - b.priority;
+        return location != 0
+          ? location
+          : a.id < b.id
+          ? -1
+          : a.id > b.id
+          ? 1
+          : a.selector < b.selector
+          ? -1
+          : a.selector > b.selector
+          ? 1
+          : 0;
```

Comment:
Not very readable, but then again neither was the original code. :man_shrugging: